### DISCUSSION TITLE
How can I make it customized

### DISCUSSION THREAD

**Discussion by cater111** [coder/code-server#2956] [Q&A] [answered]
How can I make it customized

**Comment by oxy** [comment]
What exactly do you mean by customization? You can install custom themes and extensions just like you can on VSCode.

**Comment by nooobcoder** [comment]
Please be clearer in your question @cater111 .

**Reply by nooobcoder** [reply]
@cater111 please reply to the thread, please write down your query in a more detailed way to guide you through.

**Comment by vital987** [comment, marked as answer]
Have a look at my [customization](https://github.com/vital987/code-server).
